### ISSUE TITLE
Smooth White Matter and Grey Matter contours in 3D and 2D visualizations

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#242]
Client is asking to increase the resolution of the 2D slices and 3D volumes to avoid 'pixelated' effect. I am assuming this would come from @stephenlenzi first, then we would upload it in the app @afonsobspinto

**Comment by zsinnema** [comment]
@neuroBazzi correct, if the resolution if upscaled then it will take about 2h (including testing) for @afonsobspinto

**Comment by neuroBazzi** [comment]
@stephenlenzi correct me if I am wrong, but what I understood from the client is that any (i.e. all) populations are resident and can we toggled on/off. I recalled I asked this question during the meeting but I dont' recall the answer.

**Comment by afonsobspinto** [comment]
We need to have the new upscaled images for grey and white matter, canal, grid and laminas (contour, dashed and filled)
We have django management commands to generate those (generate_annotation_images.py, generate_canal_images.py, generate_grid_images.py, generate_lamina_images.py), and what I tried to do was to adapt those according to a set of scripts @stephenlenzi shared with me:
[scripts.zip](https://github.com/MetaCell/salk-interactive-atlas/files/11668472/scripts.zip)

This issue is still pending because currently we are facing an issue, most likely caused by the upscaling of the atlas, that made the canal data disappear. @stephenlenzi already got informed about the problem. 

Branch [feature/243](https://github.com/MetaCell/salk-interactive-atlas/tree/feature/243) contains the latest developments for this issue. (not a typo, I did it alongside my work for #243)
I added  a couple of comments and refactored the code a bit to make it easier for new eyes to understand it.
Be aware that I did not had the chance to properly test the refactored code.

**Comment by stephenlenzi** [comment]
I believe I have figured out the source of the problem, there was indeed a bug in the upscaling function that meant that the canal would not be present on a subset of annotated images after upscaling. This bug would have affected some of the early manually annotated images (C1-4) but also seems to have affected all of the images with only 3 labels. The solution is a minor fix and i'll push the changes asap

**Comment by stephenlenzi** [comment]
The fix is on the contours branch see [this commit](https://github.com/MetaCell/cordmap/commit/e55f22d0fd87ca613cb54684f4a3cca731edf359) for the updated code

**Comment by neuroBazzi** [comment]
@afonsobspinto check this out
